semcore/notice/src/Notice.jsx

```diff
@@ -122,6 +122,16 @@ function Label({ styles, theme, resolveColor }) {
   return sstyled(styles)(<SLabel render={Box} use:theme={useTheme} color={color} />);
 }
 
+function Title({ styles }) {
+  const STitle = Root;
+  return sstyled(styles)(<STitle render={Box} />);
```

Comment:
I think, here should be `render={Text}` (Text - is from typography)

Reply:
Changed